$MirOS: ports/audio/faad/patches/patch-common_mp4ff_mp4ff_int_types_h,v 1.2 2006/06/25 13:07:15 bsiegert Exp $
--- common/mp4ff/mp4ff_int_types.h.orig	Sat Dec 13 23:26:34 2003
+++ common/mp4ff/mp4ff_int_types.h	Fri Nov 19 16:13:49 2010
@@ -15,7 +15,7 @@ typedef unsigned __int64 uint64_t;
 
 #else
 
-#include <stdint.h>
+#include <inttypes.h>
 
 #endif
 
